### PR TITLE
fix: only add packages directory

### PR DESCRIPTION
**Pull request by mr-tz**

### CODE REVIEW

File: .github/workflows/new_package.yml [hunk 1]
```diff
@@ -40,6 +40,8 @@ jobs:
         with:
           title: ':robot: Add ${{env.pkg_name}}.vm'
           body: 'Automated PR from ${{github.event.issue.html_url}} :rocket:'
+          add-paths:
```
Comment (Ana06) [inline .github/workflows/new_package.yml:43]: As there is only one path, I would put it in the same line

File: .github/workflows/new_package.yml [hunk 2]
```diff
@@ -40,6 +40,8 @@ jobs:
...
+            packages/*
```
Comment (mr-tz) [inline .github/workflows/new_package.yml:44]:
```suggestion
          add-paths: packages/*
```

**Review by Ana06 — APPROVED** [review]

**Comment by Ana06** [comment]
wait, we should also remove the 
```
# remove tmp file
rm pkg.json
```

That's not needed anymore. Sorry didn't realised the first time I checked it.

**Comment by mr-tz** [comment]
Good point, handled in [361c1f1](https://github.com/mandiant/VM-Packages/pull/74/commits/361c1f18009a46eb38efac9ef14eb9291cda7a2d)

**Review by Ana06 — APPROVED** [review]
thanks!

**Pull request merged**